tools/esbuild.js

```diff
@@ -0,0 +1,50 @@
+import { build } from 'esbuild';
+import { exec } from 'pkg';
+import esbuildPLuginPino from 'esbuild-plugin-pino';
+import fs from 'node:fs/promises';
+
+await build({
+  entryPoints: { 'containerbase-cli': './src/cli/index.ts' },
+  bundle: true,
+  platform: 'node',
+  target: 'node18',
+  minify: false,
+  tsconfig: 'src/cli/tsconfig.json',
+  // format: "esm", // not supoorted
```

Comment:
What's the issue?

Reply:
after compiling with pkg node will complain about dynamic import only allowed on esm. 

- https://github.com/vercel/pkg/issues/1291

Reply:
True, but you can write ESM and have it converted to CJS by esbuild. I followed [this comment][1] to get it working in [my prototype][2]

[1]: https://github.com/vercel/pkg/issues/1291#issuecomment-1518694260
[2]: https://github.com/JamieMagee/containerbase-cli

Reply:
I've written as esm and esbuild is compiling to cjs by default because of `node18` target